examples/encoding/live/start_live_encoding.py

```diff
@@ -0,0 +1,201 @@
+    video_muxing_1080p = FMP4Muxing(segment_length=4,
+                                    segment_naming='seg_%number%.m4s',
+                                    init_segment_name='init.mp4',
+                                    streams=[video_muxing_stream_1080p],
+                                    outputs=[video_muxing_1080p_output])
+    video_muxing_1080p = bitmovin.encodings.Muxing.FMP4.create(object_=video_muxing_1080p,
+                                                               encoding_id=encoding.id).resource
+    video_muxing_720p_output = EncodingOutput(output_id=s3_output.id,
+                                              output_path=OUTPUT_BASE_PATH + 'video/720p/',
+                                              acl=[acl_entry])
+    video_muxing_720p = FMP4Muxing(segment_length=4,
+                                   segment_naming='seg_%number%.m4s',
+                                   init_segment_name='init.mp4',
+                                   streams=[video_muxing_stream_720p],
+                                   outputs=[video_muxing_720p_output])
+    video_muxing_720p = bitmovin.encodings.Muxing.FMP4.create(object_=video_muxing_720p,
+                                                              encoding_id=encoding.id).resource
+
+    video_muxing_360p_output = EncodingOutput(output_id=s3_output.id,
+                                              output_path=OUTPUT_BASE_PATH + 'video/360p/',
+                                              acl=[acl_entry])
+    video_muxing_360p = FMP4Muxing(segment_length=4,
+                                   segment_naming='seg_%number%.m4s',
+                                   init_segment_name='init.mp4',
+                                   streams=[video_muxing_stream_360p],
+                                   outputs=[video_muxing_360p_output])
+    video_muxing_360p = bitmovin.encodings.Muxing.FMP4.create(object_=video_muxing_360p,
+                                                              encoding_id=encoding.id).resource
+
+    audio_muxing_output = EncodingOutput(output_id=s3_output.id,
+                                              output_path=OUTPUT_BASE_PATH + 'audio/',
+                                              acl=[acl_entry])
+    audio_muxing = FMP4Muxing(segment_length=4,
+                              segment_naming='seg_%number%.m4s',
+                              init_segment_name='init.mp4',
+                              streams=[audio_muxing_stream],
+                              outputs=[audio_muxing_output])
+    audio_muxing = bitmovin.encodings.Muxing.FMP4.create(object_=audio_muxing,
+                                                         encoding_id=encoding.id).resource
+
+    resource_response = bitmovin.encodings.Encoding.start_live(encoding_id=encoding.id, stream_key=STREAM_KEY)
+
+    bitmovin.encodings.Encoding.wait_until_running(encoding_id=resource_response.resource.id)
+
+    MAX_RETRIES = 10000
```

Comment:
fixed in https://github.com/bitmovin/bitmovin-python/pull/2/commits/e85cd368cf7981d19011573d1030aa6422ef67c7